dashboard/app/controllers/lti/v1/account_linking_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Lti
+  module V1
+    class AccountLinkingController < ApplicationController
+      before_action :lti_account_linking_enabled?
+
+      # GET /lti/v1/account_linking/landing
+      def landing
+      end
+
+      # GET /lti/v1/account_linking/existing_account
+      def existing_account
+        @user = User.new_with_session(ActionController::Parameters.new, session)
+      end
+
+      # POST /lti/v1/account_linking/link_email
+      def link_email
+        head :bad_request unless PartialRegistration.in_progress?(session)
+        params.require([:email, :password])
```

Comment:
If these params are missing, it will throw an exception. Do we care about handling that exception and letting the user know they have a missing param, or are we ok with just returning `:unauthorized` in the `else` block?

Reply:
Yeah we'll definitely want to notify the user, but I was thinking we can be added later when we finalize the UI page that will post to this route. The front-end should be able to handle this with the usual "X field is required" messaging.

Reply:
We'll also need to gracefully handle unauthorized responses for incorrect passwords